api/api_ethereum.go

```diff
@@ -727,8 +757,22 @@ func newEthRPCTransaction(tx *types.Transaction, blockHash common.Hash, blockNum
 		result.TransactionIndex = (*hexutil.Uint64)(&index)
 	}
 
-	// TODO-Klaytn: Have to add additional fields for ethereum transaction types.
-
+	switch typeInt {
+	case types.TxTypeAccessList:
+		al := tx.AccessList()
+		result.Accesses = &al
+		result.ChainID = (*hexutil.Big)(tx.ChainId())
+	case types.TxTypeDynamicFee:
+		al := tx.AccessList()
+		result.Accesses = &al
+		result.ChainID = (*hexutil.Big)(tx.ChainId())
+		result.GasFeeCap = (*hexutil.Big)(tx.GasFeeCap())
+		result.GasTipCap = (*hexutil.Big)(tx.GasTipCap())
+		// TODO-Klaytn: If we change the gas price policy from fixed to dynamic,
+		// We should recalculate gasPrice as effectiveGasPrice like below.
+		// price := math.BigMin(new(big.Int).Add(tx.GasTipCap(), baseFee), tx.GasFeeCap())
+		// result.GasPrice = (*hexutil.Big)(price)
```

Comment:
Is it okay that the APIs doesn't return `gasPrice`? I think the Ethereums gas price setting logic with `params.BaseFee` is good for this function.

Reply:
Oh, gas price was already set with `tx.GasPrice()`...  However, I think finishing the implementation is better.

Reply:
@aidan-kwon 
Applied your comment :)